src/QMCDrivers/DMC/DMC.cpp

```diff
@@ -69,7 +69,9 @@ DMC::DMC(MCWalkerConfiguration& w,
 void DMC::resetUpdateEngines()
 {
   ReportEngine PRE("DMC", "resetUpdateEngines");
-  bool fixW = (Reconfiguration == "yes");
+  bool fixW = (Reconfiguration == "runwhileincorrect");
+  if(Reconfiguration != "no" && Reconfiguration != "runwhileincorrect")
+    APP_ABORT("Reconfiguration is currently broken.  Please try again without reconfiguration.")
```

Comment:
Better to be more verbose for people who run performance test manually using old input file and newer code. They can be computer engineers.
"Reconfiguration is currently broken and gives wrong results. Please remove reconfiguration in the input section. For performance tests, please set reconfiguration to \"runwhileincorrect\" instead of \"yes\" to restore consistent behaviour."

Reply:
This is a fair comment. 

"Reconfiguration is currently broken and gives incorrect results. Set reconfiguration=no or remove the reconfiguration option from the DMC input section. To run performance tests, please set reconfiguration to "runwhileincorrect" instead of "yes" to restore consistent behaviour."

Reply:
Done.

Reply:
Sorry, didn't see Paul's comment.  Updating.

Reply:
Done.